src/Microsoft.Management.Deployment/AddPackageCatalogOptions.h

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+#include "AddPackageCatalogOptions.g.h"
+#include "public/ComClsids.h"
+
+namespace winrt::Microsoft::Management::Deployment::implementation
+{
+    [uuid(WINGET_OUTOFPROC_COM_CLSID_AddPackageCatalogOptions)]
+    struct AddPackageCatalogOptions : AddPackageCatalogOptionsT<AddPackageCatalogOptions>
+    {
+        AddPackageCatalogOptions() = default;
+
+        hstring Name();
+        void Name(hstring const& value);
+
+        hstring SourceUri();
+        void SourceUri(hstring const& value);
+
+        hstring Type();
+        void Type(hstring const& value);
+
+        winrt::Microsoft::Management::Deployment::PackageCatalogTrustLevel TrustLevel();
+        void TrustLevel(winrt::Microsoft::Management::Deployment::PackageCatalogTrustLevel const& value);
+
+        hstring CustomHeader();
+        void CustomHeader(hstring const& value);
+
+        bool Explicit();
+        void Explicit(bool const& value);
+
+        bool AcceptSourceAgreements();
+        void AcceptSourceAgreements(bool const& value);
+
+#if !defined(INCLUDE_ONLY_INTERFACE_METHODS)
+    private:
+        std::wstring m_name = L"";

```

Comment:
The pattern of using std::wstring for private fields storing strings is consistent across various APIs  *InputOptions runtimeclass implementation. However, the set and get methods use hstring. 

I can change it to hstring, but it may look inconsistent with the existing code pattern.